### PR TITLE
Callout improvments, counts up

### PR DESCRIPTION
**Pull request by Mallen220**
https://github.com/FRCTeam1987/teambroncobots.com/commit/af3a7f59e29e00231ef01c3937e49b11f27261b3

### CODE REVIEW

File: _includes/callouts.html [hunk 1]
```diff
@@ -5,35 +5,78 @@
...
+    document.addEventListener('DOMContentLoaded', function () {
+        const countUpElements = document.querySelectorAll('[data-final-count]');
+
+        const startCountUpAnimation = (entries, observer) => {
+            entries.forEach((entry) => {
+                if (entry.isIntersecting) {
+                    const element = entry.target;
+                    const finalCount = parseInt(element.getAttribute('data-final-count').replace(/,/g, ''));
+                    const minDuration = 2500; // 2.5 seconds
+                    const maxDuration = 5000; // 5 seconds
```
Comment (j0n5m1th) [inline _includes/callouts.html:57]: These can be moved out before the `forEach` (between lines 51 and 52).

File: _includes/callouts.html [hunk 2]
```diff
@@ -5,35 +5,78 @@
...
+                    const randomDuration = Math.random() * (maxDuration - minDuration) + minDuration;
```
Comment (j0n5m1th) [inline _includes/callouts.html:58]: Does this need to happen each iteration or can it happen once for all iterations?
Is it OK if this is a decimal or does it need to be a whole number?

**Review by j0n5m1th — COMMENTED** [review]
Can you please link to the dev site where this new animation is used so we can see it?

**Comment by j0n5m1th** [comment]
What branch was this created from?  It should have been merged off of `master`.  There should not be commits from 7 months ago in this PR.

**Comment by Mallen220** [comment]
> Can you please link to the dev site where this new animation is used so we can see it?

How do you want me to have this and Navbar on the dev site? Heres a video to it for now: https://drive.google.com/file/d/1MvTOLV9ykGUoJF-s5hzJf-pY97JnSGQ1/view?usp=sharing

File: _includes/callouts.html (hunk 2 above)
Comment (Mallen220) [reply _includes/callouts.html:58]: The final count and random duration values should be different for every element, the duration values can be moved. https://github.com/FRCTeam1987/teambroncobots.com/commit/20b10936ff3fc738bd77f749d1803b615dc3bf18

**Comment by j0n5m1th** [comment]
> > Can you please link to the dev site where this new animation is used so we can see it?
> 
> How do you want me to have this and Navbar on the dev site? Heres a video to it for now: https://drive.google.com/file/d/1MvTOLV9ykGUoJF-s5hzJf-pY97JnSGQ1/view?usp=sharing

This is good.  Please also do this for future changes.  It helps reviewers identify what to check!

File: _includes/callouts.html [hunk 3]
```diff
@@ -5,35 +5,78 @@
...
+                        </a>
+                        {% endif %}
+                    </div>
                     {% endfor %}
                 </div>
             </div>
         </div>
     </section>
+
+<script>
```
Comment (j0n5m1th) [inline _includes/callouts.html:41]: Can this be moved to a JS file and included in this HTML file?

Comment (Mallen220) [reply _includes/callouts.html]: https://github.com/FRCTeam1987/teambroncobots.com/commit/d68eb5fcb5a2d595564218a092be624c81db6a2b

**Review by j0n5m1th — APPROVED** [review]

**Pull request merged**